cg-spec/editors_draft.html

```diff
@@ -408,13 +408,64 @@ <h2>Parsing</h2>
+    <p>Beginning the <a href="#grammar-production-annotation">`annotation`</a> production records the |curSubject| and |curPredicate|, and sets the |curSubject| to the <a>RDF-star triple</a> |curSubject| |curPredicate| |curObject|. Finishing the <a href="#grammar-production-annotation">`annotation`</a> production restores the recorded values of |curSubject| and |curPredicate|.</p>
 
-      <p>All other productions MUST be handled as specified by <a data-cite="TURTLE#h2_sec-parsing">Section 7 of the Turtle specification</a> [[TURTLE]], while still applying the changes above recursively.</p>
+    <p>All other productions MUST be handled as specified by <a data-cite="TURTLE#h2_sec-parsing">Section 7 of the Turtle specification</a> [[TURTLE]], while still applying the changes above recursively.</p>
+
+      <section class="informative">
+      <h2>Discussion</h2>
+
+      <p class="note">This section describes parser behavior when parsing <a>embedded triples</a> and <a href="#grammar-production-annotation">annotations</a> in a Turtle-star document.</p>
+
+      <p>Turtle-star document using the <a href="#grammar-production-embTriple">`embTriple`</a> syntax results in an RDF-star graph that can be expressed using <a href="#n-triples-star">N-Triples-star</a>, as a representation of the RDF-star <a href="#concepts">abstract syntax</a>.</p>
+
+      <p>Consider a Turtle-star document that describes a single RDF triple, and also uses that triple as an <a>embedded triple</a> as the subject of another RDF-star triple:</p>
+
+      <pre class="example" title="Turtle-star embedded triples"
+           data-transform="updateExample"
+           data-content-type="text/turtle">
+      <!--
+        @base <http://example.org/> .
+        @prefix : <#> .
+
+        _:a :name "Alice" .
+        << _:a :name "Alice" >> :statedBy :bob .
+      -->
+      </pre>
+
+      <p>The usual process of parsing a Turtle document applies with the addition of matching the embedded triple `&lt;&lt; _:a :name "Alice" >>` as part of the <a href="#grammar-production-subject">`subject`</a> production. The resulting RDF-star graph can be represented in its fully expanded form in N-Triples-star:</p>
+
+      <pre class="example" title="N-Triples-star embedded triples"
+           data-transform="updateExample"
+           data-content-type="text/turtle">
+      <!--
+        _:a <http://example.org/#name> "Alice" .
+
+        << _:a <http://example.org/#name> "Alice" >>
+          <http://example.org/#statedBy> <http://example.org/#bob/> .
+      -->
+      </pre>
+
+      <p>Because the above example includes the triple `&lt;&lt; _:a :name "Alice" >>` as an RDF triple, the Turtle-star annotation syntax may also be used to represent the same RDF-star graph:</p>
```

Comment:
```suggestion
      <p>Because the above example includes the triple (|b|, `http://example.org/#name`, `"Alice"`) as an asserted triple, the same RDF-star graph may also be represented by using the Turtle-star annotation syntax as follows:</p>
```